src/EquipmentEditor.cpp

```diff
@@ -730,7 +737,9 @@ void EquipmentEditor::updateCheckboxRecord()
    {
       double bar = calcBatchSize();
       lineEdit_boilSize->setText(bar);
+      lineEdit_boilSize->setReadOnly(true);
    }
+   else lineEdit_boilSize->setReadOnly(false);
```

Comment:
On Thu, 28 Jan 2016 19:30:16 -0800
Philip Lee <notifications@github.com> wrote:

> >     }
> > +   else lineEdit_boilSize->setReadOnly(false);
> 
> These should be `setEnabled(false/true)` so that they are visually grayed out.
> 


Personally I disagree. A disabled EditText makes it look as if the value is not important in this context. To quote from a MSDN forum post.

"Use ReadOnly when you have data that you want the user to see and copy, but not modify. Use a disabled textbox, when the data you are displaying is not applicable in for the current state of a dialog or window."

However, I will change it if you insist.

Mike E

-- 
PGP key:
http://pgp.mit.edu:11371/pks/lookup?op=get&search=0x00CDB13500D7AB53